backend/app.go

```diff
@@ -193,7 +193,10 @@ func StartupApp(appName, displayAppName, appVersion, appVersionTag, latestReleas
 		ipc.DestroyConn() // cleanup socket possibly orphaned by crashed process
 		listener, err := ipc.Listen()
 		if err == nil {
-			a.ipcServer = ipc.NewServer(a.PlaybackManager, a.callOnReactivate,
+			a.ipcServer = ipc.NewServer(
+				a.PlaybackManager,
+				&a.ServerManager.Server,
```

Comment:
The IPC server should be passed just the ServerManager, as that is constant through the lifetime of the app. The Server property can change if the user logs out and logs into a different server, so it should use the fresh value of serverManager.Server for every search operation. Keep in mind the user may be logged out of all servers (UI is awaiting the login prompt, in this case servermanager.Server will be **nil**, so make sure to check to avoid nil ptr crashes)